Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/UpdateDescription.swift

```diff
@@ -44,7 +55,34 @@ public final class UpdateDescription: Codable {
         }
 
         return updateDocument
-    }()
+    }
+
+    /**
+     Unilaterally merge an update description into this update description.
+     - parameter otherDescription: the update description to merge into this
+     - returns: this merged update description
+     */
+    @discardableResult
+    public func merge(with otherDescription: UpdateDescription) -> UpdateDescription {
+        self.updatedFields = self.updatedFields.filter {
```

Comment:
Is this necessary? Will line 75 not just overwrite it?

Reply:
this is different, this is to remove fields from updatedFields that were removed by the new update description. Line 75 adds the updates from the new update description while keeping the old ones that weren't part of the new removed fields